### PR TITLE
Comprehensive fileFormat agg (fixes HumanCellAtlas/data-browser#318)

### PR DESCRIPTION
**Pull request by achave11**
Obtain fields for formation of fileFormat aggregate from one aggregate.
Added log for ES request of /summary endpoint.

### CODE REVIEW

File: src/azul/service/responseobjects/elastic_request_builder.py [hunk 1]
```diff
@@ -130,6 +130,9 @@ def create_aggregate(filters, facet_config, agg):
             field=_field,
             size=99999)
         aggregate.bucket('untagged', 'missing', field=_field)
+        if agg == "fileFormat":
+            aggregate.aggs['myTerms'].metric('size_by_type', 'sum', field="contents.files.size")
```
Comment (hannes-ucsc) [inline src/azul/service/responseobjects/elastic_request_builder.py]:
```suggestion
            fileSizeField = request_config['translation']['fileSize']
            aggregate.aggs['myTerms'].metric('size_by_type', 'sum', field=fileSizeField)
```

File: src/azul/service/responseobjects/elastic_request_builder.py [hunk 2]
```diff
@@ -130,6 +130,9 @@ def create_aggregate(filters, facet_config, agg):
...
+            aggregate.aggs['untagged'].metric('size_by_type', 'sum', field="contents.files.size")
```
Comment (hannes-ucsc) [inline src/azul/service/responseobjects/elastic_request_builder.py]:
```suggestion
            aggregate.aggs['untagged'].metric('size_by_type', 'sum', field=fileSizeField)
```

File: src/azul/service/responseobjects/hca_response_v5.py [hunk 3]
```diff
@@ -313,7 +313,7 @@ class SummaryResponse(BaseSummaryResponse):
     def __init__(self, raw_response):
         super().__init__(raw_response)
 
-        _sum = raw_response['aggregations']['by_type']
+        _sum = raw_response['aggregations']['fileFormat']["myTerms"]
```
Comment (hannes-ucsc) [inline src/azul/service/responseobjects/hca_response_v5.py]: Make sure this is referencing the sum of the file sizes and not the number of files.

**Comment by achave11** [comment]
Superseded by #536

**Pull request closed without merging**